Gopkg.lock

```diff
@@ -3,31 +3,31 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:78e80602d71837c404478f9fc6ff0a10e354b990261a5c8f05eccc6ed347614d"
+  digest = "1:6ab7a72d995ed7cd18df2ed88733e3f1ccb8004f88bdc206442df73e088a82c6"
   name = "github.com/Sirupsen/logrus"
   packages = ["."]
   pruneopts = "UT"
-  revision = "d7b6bf5e4d26448fd977d07d745a2a66097ddecb"
+  revision = "5521996833c095f5ac75d99687cef8ad344ded12"
 
 [[projects]]
-  digest = "1:73efbb654761e7e4087e4d9b912cfca51aed11329ab31888a377b46e1a3e065e"
+  digest = "1:c26c6f48d8add3ddfe9b3abf7b5d8fe2eb8f3d59f3e31e59f4488a23663e8b6b"
   name = "github.com/bitrise-io/bitrise"
   packages = ["configs"]
   pruneopts = "UT"
-  revision = "5d52fad1a1fbe34417296f1fbf6d3672c5d5efc9"
-  version = "1.27.1"
+  revision = "a7523d99944aaf56c644dec41f374f746db9a453"
+  version = "1.29.0"
```

Comment:
for some reason "github.com/bitrise-io/bitrise" is not part of the `Gopkg.toml`, could you please add it and lock for the `master` branch